_tech-blog/2020-08-31-gsoc-20-resume.markdown

```diff
@@ -0,0 +1,61 @@
+---
+title: GSoC 20 - Automatized support for Organised Editing Guidelines
+date: 2020-08-31 12:30:00 Z
+Summary Text: Here is the resume from the GSoC' 20 project that adds
+  automatized support for Organised Editing Guidelines in
+  Tasking Manager.
+Feature Image: "/uploads/gsoc-20-architecture.jpg"
+Person:
+- Wille Marcel
+---
+
+
+# GSoC 20 - Automatized support for Organised Editing Guidelines
+
+## The problem
+&nbsp;&nbsp;&nbsp;&nbsp;The OpenStreetMap (OSM) project has established formal [Organised Editing Guidelines](https://wiki.osmfoundation.org/wiki/Organised_Editing_Guidelines), which require all groups and organized activities of mapping on OpenStreetMap to report transparently how they organize and what they are mapping on.
+&nbsp;&nbsp;&nbsp;&nbsp;The Tasking Manager is the most used application in the OSM ecosystem for volunteers, but also professionals, to team up and coordinate mapping. In order to ease up the user experience of fulfilling the guidelines for organized editing, the goal of my project this summer is to make the process of reporting back data for OpenStreetMap automatically.
+
+## Project solutions
+
+&nbsp;&nbsp;&nbsp;&nbsp;In the first step, we started with an analysis of contents and fields that are required or suggested by the organized editing guidelines, then we check whether these are already available in the Tasking Manager, or not.
+&nbsp;&nbsp;&nbsp;&nbsp;Then we implemented a small proof of concept for testing out how the communication between [Tasking Manager](https://tasks.hotosm.org/) and OSM Community should occur. Although the OSM’s organized editing guidelines suggest the project wiki for reporting of organized mapping activities, we wanted to include two technical ideas for this initial test:
+- Report data directly to the [OSM wiki](https://wiki.openstreetmap.org/wiki/Main_Page)
+- Store files in a human-readable format in a git repository (GitHub, Gitlab, etc)
+
+## Validating ideas
+&nbsp;&nbsp;&nbsp;&nbsp;We participated in the [State of the Map 2020](https://2020.stateofthemap.org/) conference as a self-organized session in order to validate the project solutions initially thought and receive more feedback from the community.
+&nbsp;&nbsp;&nbsp;&nbsp;We received some suggestions on the initial ideas, mainly for reporting data to the wiki directly instead of reporting to a git repository. After discussing internally, we reached a consensus that it would be better to use both solutions, instead of just one as thought at the beginning of the project. Comming to this architecture and also, a cool name to the project - [OEG Reporter](https://github.com/hotosm/oeg-reporter). <img src="https://user-images.githubusercontent.com/18130942/91627404-7ae96c80-e98d-11ea-98a7-999c754cd19f.png" />
```

Comment:
always let one blank line between text and images

Reply:
I don't know if it's ok to upload the images to github cdn... I've already lost some images because of that. @dakotabenjamin some opinion on it?

Reply:
Removed the images from the GitHub CDN and uploaded the images to the repository. Also, added one blank line between text and images